src/library.js

```diff
@@ -3208,13 +3208,15 @@ mergeInto(LibraryManager.library, {
   },
 
 #if DYNCALLS || !WASM_BIGINT
+#if MAIN_MODULE == 1
+  // createDyncallWrapper is defined in library_makeDynCall
```

Comment:
```suggestion
```
(not sure that helps much here - a grep can easily find it, and if we refactor code around we might forget to update this)